tests/TestCase/Database/QueryTest.php

```diff
@@ -1431,10 +1431,10 @@ public function testWhereWithBetween()
 
         $this->assertCount(2, $result);
         $first = $result->fetch('assoc');
-        $this->assertEquals(5, $first['id']);
+        $this->assertSame('5', $first['id']);
```

Comment:
Is the string here really what we want? 
Doesnt the ORM cast this to int as per field type?

Reply:
It does but it needs the field type map for that. Since `Database/Query` is directly used here and the type map hasn't been explicitly set, it can't do any conversion, hence string is expected.